Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -131,4 +131,33 @@ public string SanitizeMessageCapital(string message)
         message = char.ToUpper(message[0]) + message.Remove(0, 1);
         return message;
     }
+
+    public string SanitizeMessageCapitalizeTheWordI(string message, string theWordI = "i")
+    {
+        if (string.IsNullOrEmpty(message))
+            return message;
+
+        for
+        (
+            var index = message.IndexOf(theWordI);
+            index != -1;
+            index = message.IndexOf(theWordI, index + 1)
+        )
+        {
+            // Stops the code If It's tryIng to capItalIze the letter I In the mIddle of words
+            // Repeating the code twice is the simplest option
+            if (index + 1 < message.Length && char.IsLetter(message[index + 1]))
+                continue;
+            if (index - 1 >= 0 && char.IsLetter(message[index - 1]))
+                continue;
+
+            var beforeTarget = message.Substring(0, index);
+            var target = message.Substring(index, theWordI.Length);
+            var afterTarget = message.Substring(index + theWordI.Length);
+
+            message = beforeTarget + target.ToUpper() + afterTarget;
+        }
+
+        return message;
+    }
```

Comment:
Can't you just split and check?

Also this doesn't seem very localisation friendly, better that it's more generic instead.

Reply:
I think to use splits because it didn't occur to me until after everything was finished. I think code using splits wouldn't be a whole lot shorter than what I have here.

As for locale, something I just learned is that English is luckily the only language to capitalize I, so unless this function is expected to be used to capitalize arbitrary quantities of words of arbitrary length or something (at which point it would be different enough to be a separate PR), the lack of genericness shouldn't be a problem.

If either of those two paragraphs are wrong then I'll have to change something, but otherwise things should be good